enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,325 @@
+
+## Summary
+
+To enable automated backups of Etcd database and cluster resources of an Openshift cluster from day one after installation, 
+without provided configuration from the user.
+
+This document outlines the possible approaches that were investigated, discussed and tested, in addition to the final implemented approach.  
+
+## Motivation
+
+The current method of **Backup** and **Restore** of an Openshift cluster is outlined in  [Automated Backups of etcd](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md). 
+This approach relies on user supplied configuration, in the form of `config.openshift.io/v1alpha1 Backup` CR. This method utilizes `batchv1.CronJob` in order to take Backups of Etcd and cluster resources.
+
+To improve the user's experience, this work proposes taking etcd backup using default configurations from day one, without user interference.
+
+Moreover, for recovery in disaster scenarios, where more than one control-plane node is lost from an Openshift cluster, this proposal attempts to store backups on all control-plane nodes.
+Having backups of an Openshift cluster on all control-plane nodes, improves the recovery probability in disaster scenarios.
+
+### User Stories
+
+* As a cluster administrator, I would like an OCP cluster backups to be taken without user's supplied configuration from day one, after installation.
+* As a cluster administrator, I would like to an OCP cluster backups available on all master nodes, so that I have a recent cluster state to recover from in the event of losing control-plane nodes.
+
+### Goals
+
+* Backups should be taken without configuration after cluster installation from day 1.
+* Backups are saved to a default location on each master node's file system.
+* Backups are taken according to a default schedule.
+* Backups are being maintained according to a default retention policy.
+
+### Non-Goals
+
+* Save cluster backups to remote cloud storage (e.g. S3 Bucket).
+  - This could be a future enhancement or extension to the API.
+  - Having backups in an independent storage from OCP cluster is crucial for Disaster Recovery scenarios.
+* Automate cluster restoration.
+* Provide automated backups for non-self hosted architectures like Hypershift.
+
+## Proposal
+
+This work utilizes the [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) from current `CronJob` based implementation.
+Moreover, it uses the same [PeriodicBackupController](https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go#L81) as this proposal builds upon it, and also under the same feature gate.
+The default configuration is created through `config.openshift.io/v1alpha1 Backup` CR, with name `default`, to distinguish it from other user supplied `Backup` CRs.
+Using a CR enable users to override the default backup configuration.
```

Comment:
Does *not* having a CR named `default` result in a default configuration? If we plan to have a default CR, can that CR be deleted? What if I enable the CronJob backups featurre but not the default backups, will I be prevented from creating a CR named 'default'?

Reply:
I don't know that it will be immediately obvious that a CR with a special name should behave differently than other CRs of the same type.
IMO it might make sense to create a separate configuration CRD for this (could share most of its spec with the current CRD)

Reply:
good points, Cesar. I think I was driven mostly by the special case "default" namespace, which is a similar kind of special case.

Reply:
If we use the installer to generate the default Backup CR, and then have it applied during cluster bootstrap. That removes any logic about it being a special name, and allows users to delete and remove it should they wish.

I believe the current approach however is a create only CR inside the payload, which means if you deleted the CR, it would come back